### PR TITLE
[FIX] redirects: add redirections for moves in accounting

### PR DESCRIPTION
**Pull request by jcs-odoo**
- receivables/customer_invoices/refund --> receivables/customer_invoices/credit_notes
- receivables/customer_payments/payment_sepa --> receivables/customer_payments/batch_sdd
- overview/main_concepts/intro accounting
- bank/feeds/synchronize --> bank/feeds/bank_synchronization
- others/adviser/assets --> payables/supplier_bills/assets
- others/configuration/account_type --> overview/getting_started/chart_of_accounts
- bank/feeds/manual --> bank/feeds/bank_statements
- bank/feeds/coda --> bank/feeds/bank_statements
- bank/feeds/ofx --> bank/feeds/bank_statements
- bank/feeds/qif --> bank/feeds/bank_statements
- receivables/customer_invoices/installment_plans --> receivables/customer_invoices/payment_terms
- others/configuration --> accounting
- others/adviser/fiscalyear --> fiscality/fiscal_year/close_fiscal_year
- others/configuration/fiscal_year --> fiscality/fiscal_year/fiscal_year
- others/taxes --> fiscality/taxes
- others/taxes/B2B_B2C --> fiscality/taxes/B2B_B2C
- others/taxes/application --> fiscality/taxes/application
- others/taxes/cash_basis_taxes --> fiscality/taxes/cash_basis_taxes
- others/taxes/create --> fiscality/taxes/create
- others/taxes/default_taxes --> fiscality/taxes/default_taxes
- others/taxes/retention --> fiscality/taxes/retention
- others/taxes/tax_included --> fiscality/taxes/tax_included
- others/taxes/taxcloud --> fiscality/taxes/taxcloud

### CODE REVIEW

File: redirects.txt [hunk 1]
```diff
@@ -42,4 +42,28 @@ sales/sale_ebay.rst sales/ebay_connector.rst                    # sale_ebay -> e
 sales/ebay/setup.rst sales/ebay_connector/setup.rst             # ebay/* moved to ebay_connector/* (#524)
 sales/ebay/manage.rst sales/ebay_connector/manage.rst           # ebay/* moved to ebay_connector/* (#524)
 
-accounting/localizations/nederlands.rst accounting/localizations/netherlands.rst  # (#514)
\ No newline at end of file
+accounting/localizations/nederlands.rst accounting/localizations/netherlands.rst  # (#514)
+
+accounting/receivables/customer_invoices/refund.rst accounting/receivables/customer_invoices/credit_notes.rst     # refund --> credit_notes (#475)
+accounting/receivables/customer_payments/payment_sepa.rst accounting/receivables/customer_payments/batch_sdd.rst  # payment_sepa --> batch_sdd (#475)
+accounting/overview/main_concepts/intro.rst accounting.rst                                                        # intro --> [deleted] (#482)
```
Comment (AntoineVDV) [inline redirects.txt:49]: You don't need to handle deleted pages that have no alternative.

File: redirects.txt [hunk 2]
```diff
@@ -42,4 +42,28 @@ sales/sale_ebay.rst sales/ebay_connector.rst                    # sale_ebay -> e
...
+accounting/others/configuration/account_type.rst accounting/overview/getting_started/chart_of_accounts.rst        # others/configuration/account_type --> overview/getting_started/chart_of_accounts (#554)
+accounting/bank/feeds/manual.rst accounting/bank/feeds/bank_statements.rst                                        # manual --> bank_statements [merged] (#569)
+accounting/bank/feeds/coda.rst accounting/bank/feeds/bank_statements.rst                                          # coda --> bank_statements [merged] (#569)
+accounting/bank/feeds/ofx.rst accounting/bank/feeds/bank_statements.rst                                           # ofx --> bank_statements [merged] (#569)
+accounting/bank/feeds/qif.rst accounting/bank/feeds/bank_statements.rst                                           # qif --> bank_statements [merged] (#569)
+accounting/receivables/customer_invoices/installment_plans.rst accounting/receivables/customer_invoices/payment_terms.rst # installment_plans --> payment_terms [merged] (#573)
+accounting/others/configuration.rst accounting.rst                                                                # configuration/* --> [deleted] (#575)
```
Comment (AntoineVDV) [inline redirects.txt:58]: Idem

**Comment by jcs-odoo** [comment]
@AntoineVDV 
updated, rebased and merged but GitHub keeps saying that there is still a conflict.

**Comment by AntoineVDV** [comment]
Try running `git fetch origin && git rebase origin/13.0 && git push --force-with-lease`

**Comment by AntoineVDV** [comment]
I don't know how you did (probably commit amend) it but you pushed CDU's commit 🤔

**Comment by jcs-odoo** [comment]
@AntoineVDV 
should be good to go :)

**Pull request merged**